packages/commerce-sdk-react/setup-jest.js

```diff
@@ -6,6 +6,8 @@
  */
 import '@testing-library/jest-dom'
 import {configure} from '@testing-library/dom'
+import {networkInterfaces} from 'os'
```

Comment:
Is `networkInterfaces` an unused import or is it required internally for something ?

Reply:
Good catch, let me remove that